tools/dynamatic/dynamatic.cpp

```diff
@@ -560,7 +562,20 @@ CommandResult Compile::execute(CommandArguments &args) {
     return CommandResult::FAIL;
 
   std::string script = state.getScriptsPath() + getSeparator() + "compile.sh";
-  std::string buffers = args.flags.contains(SIMPLE_BUFFERS) ? "1" : "0";
+  std::string buffers = "simple-buffers";
+
+  if (auto it = args.options.find(BUFFER_ALGORITHM); it != args.options.end()) {
+    if (it->second == "simple-buffers" || it->second == "fpga20" ||
+        it->second == "fpl22")
+      buffers = it->second;
```

Comment:
nit pick: brackets around the "then" branch to be uniform with the "else" branch ([LLVM guidelines, 4th code example](https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements))